kqueen/engines/gce.py

```diff
@@ -143,26 +143,38 @@ def __init__(self, cluster, **kwargs):
         self.project = self.service_account_info.get('project_id', '')
         self.zone = kwargs.get('zone', '-')
         self.cluster_id = 'a' + self.cluster.id.replace('-', '')
+
+        # Generate metadata for Network Policies if empty
+        if not isinstance(cluster.metadata.get('network_policy', None), dict):
```

Comment:
`cluster.metadata.get('network_policy')`
None will be by default

Reply:
done